.buildkite/longruns_gpu/pipeline.yml

```diff
@@ -31,6 +31,16 @@ steps:
   - group: "Global Land Models"
     steps:
 
+      - label: "Snowy Land"
```

Comment:
we could add the :mountain_snow: emoji for this run :)

Reply:
nice! will do :)